x-pack/plugins/observability_solution/apm/public/components/shared/entity_enablement/welcome_modal.tsx

```diff
@@ -0,0 +1,121 @@
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiButton,
+  EuiConfirmModal,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiImage,
+  EuiLink,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { useKibanaUrl } from '../../../hooks/use_kibana_url';
+
+export function Welcome({
+  isModalVisible = false,
+  onClose,
+  onConfirm,
+}: {
+  isModalVisible: boolean;
+  onClose: () => void;
+  onConfirm: () => void;
+}) {
+  const servicesInventory = useKibanaUrl('/plugins/apm/assets/services_inventory.png');
+
+  return (
+    <>
+      {isModalVisible && (
+        <EuiConfirmModal
+          style={{
+            width: '630px',
+          }}
+          onCancel={onClose}
+          onConfirm={onConfirm}
+          confirmButtonText={
+            <EuiButton data-test-subj="xpack.apm.welcome.button.open" fill size="s">
+              {i18n.translate('xpack.apm.welcome.button.openSurvey', {
+                defaultMessage: 'OK',
+              })}
+            </EuiButton>
+          }
+          cancelButtonText={
+            <EuiLink
+              target="_blank"
+              data-test-subj="apmWelcomeLinkExternal"
+              href="https://ela.st/new-experience-services"
+              external
+            >
+              {i18n.translate('xpack.apm.welcome.linkLabel', {
+                defaultMessage: 'Learn more',
+              })}
+            </EuiLink>
+          }
+          defaultFocusedButton="confirm"
+        >
+          <EuiPanel hasShadow={false}>
+            <EuiFlexGroup
+              direction="column"
+              justifyContent="center"
+              alignItems="center"
+              gutterSize="m"
+            >
+              <EuiFlexItem>
+                <EuiIcon type="logoElastic" size="l" />
+              </EuiFlexItem>
+              <EuiFlexItem>
+                <EuiTitle>
+                  <h2>
+                    {i18n.translate('xpack.apm.welcome.title', {
+                      defaultMessage: 'Welcome to our new experience!',
+                    })}
+                  </h2>
+                </EuiTitle>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiPanel>
+
+          <EuiPanel hasShadow={false} paddingSize="s">
+            <EuiText grow={false} textAlign="center">
+              <p>
+                {i18n.translate('xpack.apm.welcome.body', {
+                  defaultMessage:
+                    'You can now see services detected from logs alongside services instrumented with APM our new service inventory so you can view all of your services in a single place.',
+                })}
+              </p>
+            </EuiText>
+          </EuiPanel>
+          <EuiSpacer size="m" />
+          <EuiPanel hasBorder paddingSize="none">
+            <EuiImage
+              size="xl"
+              src={servicesInventory}
+              alt={i18n.translate('xpack.apm.welcome.image.at,', {
```

Comment:
I guess there is an extra `,`  in the translation key. Also, perhaps we don't need `.at`
```suggestion
              alt={i18n.translate('xpack.apm.welcome.image', {
```